### ISSUE TITLE
SourceFile.formatText() should respect indentation settings

### ISSUE THREAD

**Issue by Maximaximum** [dsherret/ts-simple-ast#149]
When I run the following code, the indentationText manipulation setting gets actually ignored inside the `file.formatText();` call. The indentation gets produced as 4 space characters, no matter what is passed to the Ast constructor.

```
  const ast = new Ast({
    manipulationSettings: {
      indentationText: IndentationText.Tab
    }
  });
  ast.addSourceFiles(filesGlob);

  const files = ast.getSourceFiles();

  for (const file of files) {
    const namespaces = file.getNamespaces();
    for (const namespace of namespaces) {
      namespace.unwrap();
    }
  }

  for (const file of files) {
    file.formatText();
  }
  ast.saveUnsavedSourceFilesSync();
```

**Comment by dsherret** [comment]
Thanks for reporting this. I'm surprised the underlying typescript compiler printer will always change it to 4 spaces and doesn't have a way to customize that. When formatting, I'll just have to add an extra pass through the file in order to change the 4 spaces to the current manipulation settings.

**Comment by dsherret** [comment]
This is fixed in 1.2.0. It will also use the newline kind specified in the manipulation settings.

Let me know if you notice any issues.

**Comment by Maximaximum** [comment]
Thanks a lot! This is fixed now indeed :)
